app/Filters/ThreadFilters.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Filters;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+
+class ThreadFilters
+{
+
+    /**
+     * @var Request
+     */
+    protected $request;
+
```

Comment:
Properties should have a block explaining what they are in addition to what instance they are. e.x. 
<img width="265" alt="Screen Shot 2021-08-05 at 6 22 02 PM" src="https://user-images.githubusercontent.com/10804999/128352940-67fbf1fd-54cf-4453-8b05-dec289def3b8.png">